[2025-01-31T05:17:38.679+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v5.third_task scheduled__2023-02-13T00:00:00+00:00 [queued]>
[2025-01-31T05:17:38.699+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v5.third_task scheduled__2023-02-13T00:00:00+00:00 [queued]>
[2025-01-31T05:17:38.699+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T05:17:38.728+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): third_task> on 2023-02-13 00:00:00+00:00
[2025-01-31T05:17:38.742+0000] {standard_task_runner.py:57} INFO - Started process 19194 to run task
[2025-01-31T05:17:38.749+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v5', 'third_task', 'scheduled__2023-02-13T00:00:00+00:00', '--job-id', '5996', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpyuvu8oe9']
[2025-01-31T05:17:38.751+0000] {standard_task_runner.py:85} INFO - Job 5996: Subtask third_task
[2025-01-31T05:17:38.957+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v5.third_task scheduled__2023-02-13T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T05:17:39.404+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v5' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2023-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-13T00:00:00+00:00'
[2025-01-31T05:17:39.409+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T05:17:39.412+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the third task, I run after the task 1, at the same time with task 2']
[2025-01-31T05:17:39.439+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T05:17:39.443+0000] {subprocess.py:93} INFO - I am the third task, I run after the task 1, at the same time with task 2
[2025-01-31T05:17:39.444+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T05:17:39.540+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v5, task_id=third_task, execution_date=20230213T000000, start_date=20250131T051738, end_date=20250131T051739
[2025-01-31T05:17:39.654+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T05:17:39.709+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
